src/Calculator.php

```diff
@@ -0,0 +1,95 @@
+<?php
+namespace Money;
+
+/**
+ * Interface Calculator
+ * @package Money
```

Comment:
Can you please remove package phpdoc and add yourself as an author (If you are OK with that)?